documentation/modules/ROOT/pages/configuration/signalling.adoc

```diff
@@ -178,12 +178,25 @@ Currently {prodname} supports the `incremental` and `blocking` types.
 
 |`data-collections`
 |_N/A_
-| An array of comma-separated regular expressions that match the fully-qualified names of the tables to include in the snapshot. +
+| An array of comma-separated regular expressions that match the fully-qualified names of the data collections to include in the snapshot. +
 Specify the names by using the same format as is required for the `signal.data.collection` configuration option.
 
-|`additional-condition`
+|`[.line-through]#additional-condition#`
+|_N/A_
+| An optional string that specifies a condition that the connector evaluates to designate a subset of records to include in a snapshot. +
++
+[NOTE]
+====
+This property is deprecated and should be replaced by additional-conditions.
+====
+
+|`additional-conditions`
 |_N/A_
-| An optional string that specifies a condition that the connector evaluates to designate a subset of records to include in a snapshot.
+| An optional array of additional condition that specifies a condition that the connector evaluates to designate a subset of records to include in a snapshot. +
+  Each additional condition is an object with `data-collection` and `filter` properties. You can now specify different filters for different data collection.
+
+ * The `data-collection` property is the fully-qualified name of the data collection for which the filter will be applied.
+ * The `filter` property will have the same value used in the old property for the `incremental snapshot` and the value used in the `snapshot.select.statement.overrides` for the `blocking snapshot`.
```

Comment:
```suggestion
Each additional condition is an object that specifies the criteria for filtering the data that an ad hoc snapshot captures.
You can set the following parameters for each additional condition:

You can set the following parameters for each additional condition:

`data-collection`:: The fully-qualified name of the {data-collection} that the filter applies to.
You can apply different filters to each {data-collection}. 
`filter`:: Specifies column values that must be present in a database record for the snapshot to include it, for example,  `"color='blue'"`. +
 +
The values that you assign to the `filter` parameter are the same types of values that you might specify in the `WHERE` clause of `SELECT` statements when you set the `snapshot.select.statement.overrides` property for a blocking snapshot.
In earlier {prodname} releases, an explicit `filter` parameter was not defined for snapshot signals; instead, filter criteria were implied by the values that were specified for the now deprecated `additional-condition` parameter. 
```

Reply:
> You can set the following parameters for each additional condition:

Also these are still properties

> The values that you assign to the `filter` parameter are the same types of values that you might specify in the `WHERE` clause of `SELECT` statements when you set the `snapshot.select.statement.overrides` property for a blocking snapshot.
	In earlier {prodname} releases, an explicit `filter` parameter was not defined for snapshot signals; instead, filter criteria were implied by the values that were specified for the now deprecated `additional-condition` parameter.

This is not really true. If you are doing an incremental snapshot you can now use the filter to put what should be used in the where clause (the same used in the `additional-condition`) with the plus that now you can specify multiple filters. 

For the blocking snapshot the filters has a different meaning, you need to pass the full query statement as you can do for the initial snapshot using the `snapshot.select.statement.overrides`

Is it more clear now?